packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -198,13 +205,20 @@ class _DraggableState<T> extends State<Draggable<T>> {
 
   @override
   void dispose() {
-    _recognizer.dispose();
+    _disposeRecognizerIfInactive();
```

Comment:
please add a comment here or in _disposeRecognizerIfInactive or at the class level discussing the unusual lifetime conventions of this object and the _recognizer it creates.